test/ex_unit_properties_test.exs

```diff
@@ -33,6 +33,22 @@ defmodule ExUnitPropertiesTest do
         Enum.take(data, 1)
       end
     end
+
+    test "supports do keyword syntax" do
+      gen(all _boolean <- boolean(), do: :ok)
```

Comment:
We should add `gen` to a local `.formatter.exs` (or to Elixir one) so we can get rid of parens.

Reply:
@josevalim Elixir seems a bit too broad, but I am okay with adding it there. Do you have a preference?

Reply:
If the plan is inclusion in Elixir, then yes. Otherwise, no.